### PR TITLE
use around

### PR DESCRIPTION
**Pull request by grosser**
@sinsoku

### CODE REVIEW

**Comment by grosser** [comment]
@xaviershay @colszowka I can haz merge ?

File: spec/return_codes_spec.rb [hunk 1]
```diff
@@ -4,14 +4,8 @@
 # See https://github.com/colszowka/simplecov/issues/5
 describe "return codes" do
   context "inside fixtures/frameworks" do
-    before do
-      @current_dir = Dir.getwd
-      Dir.chdir(File.join(File.dirname(__FILE__), "fixtures", "frameworks"))
-      FileUtils.rm_rf("./coverage")
```
Comment (xaviershay) [inline spec/return_codes_spec.rb]: why is it ok to drop this line?

Comment (grosser) [inline spec/return_codes_spec.rb]: it is not, fixed!

**Comment by xaviershay** [comment]
great, thanks

**Pull request merged**